youtube_dl/extractor/iqm2.py

```diff
@@ -0,0 +1,85 @@
+# meetings.  In some cases (e.g. cambridgema.iqm2.com), after a hefty
+# (several-hour) processing time, that video is available in easily
+# downloadable form from their web portal, but prior to that, the
+# video can only be watched in realtime through JWPlayer. Other
+# (somervillecityma.iqm2.com) instances don't seem to ever offer a
+# downloadable form. This extractor is designed to download the
+# realtime video without the download links being available. For more
+# info on Accela, see:
+#   http://www.iqm2.com/About/Accela.aspx
+#   http://www.accela.com/
+#   https://github.com/Accela-Inc/leg-man-api-docs
+
+# This processing makes hard to test since there's only a narrow
+# window when it matters. After that the extractor finds links to
+# the processed video intead.
+
+# No metadata is retrieved, as that would require finding a metadata
+# URL and retreiving a 3rd HTTP resource.
+
+# Contributed by John Hawkinson <jhawk@mit.edu>, 6 Oct 2016.
+
+class IQM2IE(InfoExtractor):
+
+    # We commonly see both iqm2.com and IQM2.com.
+    _VALID_URL = r'(?i)https?://(?:\w+\.)?iqm2\.com/Citizens/\w+.aspx\?.*MeetingID=(?P<id>[0-9]+)'
+    _TESTS = [ {
+            'url': 'http://somervillecityma.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=2308',
+         'md5': '9ef458ff6c93f8b9323cf79db4ede9cf',
+         'info_dict': {
+             'id': '70472_480',
+             'ext': 'mp4',
+             'title': 'City of Somerville, Massachusetts',
+             'uploader': 'somervillecityma.iqm2.com',
+         }}, {
+        'url': 'http://cambridgema.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=1679#',
+        'md5': '478ea30eee1966f7be0d8dd623122148',
+        'info_dict': {
+            'id': '1563_720',
+            'ext': 'mp4',
+            'title': 'Cambridge, MA (2)',
+            'uploader': 'cambridgema.iqm2.com',
+        }}, {
+            'url': 'https://CambridgeMA.IQM2.com/Citizens/VideoMain.aspx?MeetingID=1679',
+            'only_matching': True,
+        }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # print "Original URL is", url
+
+        # Simple extractor: take, e.g.
+        #   http://cambridgema.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=1679
+        # and look for
+        #   <div id="VideoPanel" class="LeftTopContent">
+        #     <div id="VideoPanelInner" ... src="/Citizens/VideoScreen.aspx?MediaID=1563&Frame=SplitView">
+        # and feed the canonicalized src element to the generic extractor
+        inner_url_rel = self._html_search_regex(
+            r'<div id="VideoPanelInner".*src="([^"]+)"',
+            webpage, 'url');
+        # print "inner_URL is", inner_url_rel
+
+        inner_url = compat_urlparse.urljoin(url, inner_url_rel)
+
+        if self._downloader.params.get('verbose'):
+            self.to_screen('Invoking downloader on %s' % inner_url)
```

Comment:
Or not. I can't seem to find the exact reason that I wanted this.
I think [L1640](https://github.com/rg3/youtube-dl/blob/176006a1202cc6ef3d0a768ace41f97516c76c6d/youtube_dl/YoutubeDL.py#L1640) seems to display it under normal circumstances.

Maybe I wanted it to display when running test_download.py? I'm not sure, but that doesn't seem a good enough reason to modify `process_ie_result`.

Dropping this unless someone disagrees.